device/main.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
```

Comment:
nit: are you looking to include any license info yet? I know you mentioned it might not live in Microsoft

Reply:
I'll wait for the repo decision to include license.

---
In reply to: [543620401](https://github.com/mamokarz/sample-multi-platform/pull/2#discussion_r543620401) [](ancestors = 543620401)